internal/migrations/v2.9.0.go

```
@@ -1,17 +1,92 @@
 	"github.com/knadh/koanf/v2"
 	"github.com/knadh/stuffbin"
 )
 
+// V2_9_0 adds globally unique ownership and verification state for inbox email addresses.
 func V2_9_0(db *sqlx.DB, fs stuffbin.FileSystem, ko *koanf.Koanf) error {
-	_, err := db.Exec(`
+	tx, err := db.Beginx()
+	if err != nil {
+		return err
+	}
+	defer tx.Rollback()
```

golangci-lint (2.13.2)

[error] Error return value of `tx.Rollback` is not checked

(errcheck)